src/components/details/details.component.js

```diff
@@ -66,7 +72,7 @@ export default function Details() {
             {
               details.description &&
               <div className="item-description">
-                <p className="primary-text">{details.description}</p>
+                <p className="primary-text" dangerouslySetInnerHTML={{ __html: details.description ? DOMPurify.sanitize(marked(details.description)) : '' }}></p>
```

Comment:
already check in line 73 like  details.description && is it required to details.description ? DOMPurify.sanitize(marked(details.description)) : ''